commerce-theme-minium/commerce-theme-minium/src/css/_login.scss

```diff
@@ -168,17 +175,32 @@
                 margin-top: auto;
             }
 
-            input, label {
-                font-size: 16px;
+            input {
+                color: $color-fg;
 
-                &#_com_liferay_login_web_portlet_LoginPortlet_login {
+                &[type="text"],
+                &[type="password"] {
                     height: 55px;
-                    font-size: 20px;
                 }
 
-                &#_com_liferay_login_web_portlet_LoginPortlet_password {
+                &[type="text"] {
+                    font-size: 15px;
+                    font-weight: 500;
+                    letter-spacing: 1px;
+                }
+
+                &[type="password"] {
                     height: 55px;
-                    font-size: 33px;
+                    font-size: 50px;
+                    letter-spacing: -4px;
```

Comment:
this also could use an alpha sort, though it was already wrong before these changes